### PR TITLE
adding improvmx email

### PR DESCRIPTION
**Pull request by reqrt**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

**Comment by mdsmax** [comment]
dont need more

**Comment by mdsmax** [comment]
can you try it? i need again.

File: domains/max.json [hunk 1]
```diff
@@ -7,6 +7,11 @@
 
       "record": {
         "URL": "https://eu.dzxdev.repl.co"
```
Comment (mdsmax) [inline domains/max.json:9]: remove this, I changed to a CNAME

**Comment by WilliamDavidHarrison** [comment]
We don't support CNAME with other records.

**Review by wdhdev — APPROVED** [review]

**Pull request merged**